### PR TITLE
Check for the WooCommerce Admin class before adding notice

### LINKED ISSUE #25394: WooCommerce Admin plugin notice shown if folder name does not mach
**Describe the bug**
If wc-admin is installed in a directory not named `woocommerce-admin` the notice to install shown wc-admin will be shown.

Minor, but we can update this to check for the namespace and class in place of the folder name.

**To Reproduce**
1. Install wc-admin.
1. Rename to anything other than `woocommerce-admin`.
1. Trigger `woocommerce_installed` or switch themes to trigger the notice.
1. Note the notice.

**Screenshots**
Screen Shot 2020-01-13 at 5 38 59 PM

**Expected behavior**
Expected to not see this notice.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by joshuatf**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Check to see if WooCommerce Admin FeaturePlugin class exists before adding notice instead of checking for default plugin folder.

This is a minor issue, but can happen if the folder is not using the default name.

Screen Shot 2020-01-13 at 5 38 59 PM

Closes #25394

### How to test the changes in this Pull Request:

1. Install wc-admin.
1. Rename to anything other than woocommerce-admin.
1. Trigger `woocommerce_installed` or switch themes to trigger the notice.
1. Make sure the above notice isn't shown.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Check for WooCommerce Admin feature plugin class before adding install notice

### CODE REVIEW

**Review by rodrigoprimo — APPROVED** [review]
Looks good to me. I see that there are pending reviews assigned to @timmyc and the WooCommerce admin team. Should I wait for them to check this PR or should I proceed and merged it?

**Comment by timmyc** [comment]
All good to merge if it test out well for you @rodrigoprimo

**Pull request merged**
